src/libs/style.ts

```diff
@@ -28,18 +28,28 @@ const cardStyles: Record<string, CSSProperties> = {
     backgroundColor: 'white',
     boxShadow: '0 2px 5px 0 rgba(0,0,0,0.35), 0 3px 2px 0 rgba(0,0,0,0.12)',
   },
-};
+} as const satisfies Record<string, CSSProperties>;
 
 export const elements = {
   card: cardStyles,
```

Comment:
Unfortunate that the multiple levels of nesting here complicates the type signature.